.sync.yml

```diff
@@ -41,8 +41,10 @@ Gemfile:
         condition: 'ENV["GEM_BOLT"]'
       - gem: async
         version: '~> 1.30' # otherwise async 2.0.0(needs ruby >=3.1.0) is wrongly selected by bundler on jenkins while running with ruby 2.7.1
+    ":system_tests":
       - gem: puppet_litmus
         version: '0.34.5'
```

Comment:
Sorry, I should have looked yesterday to see if we still need to pin puppet_litmus. Looking at the ticket I filed for this with the CAT team, it doesn't seem like we do: https://perforce.atlassian.net/browse/CAT-756